app/src/main/res/values/strings.xml

```diff
@@ -846,4 +846,6 @@
     <string name="conf_error_value_to_assign_option_set">Program rule %s is trying to assign value %s which does not belongs to option set %s</string>
     <string name="conf_error_value_option_set">Value for field %s does not belong to the option set in %s</string>
     <string name="action_do_not_show_again">Do not show again</string>
+    <string name="feature_configuration">Feature Configuration</string>
```

Comment:
this string is declared in commons, please remove it

Reply:
I think transifex translations only fetch data from the main string.xml, so we need to overwrite (and duplicate) the keys in any module so that it can be translated. I have to make sure, but we may need to update how the translations are handled.

Reply:
Good catch 🧤 !